xls/public/test_c_api_symbols.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2024 The XLS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+import unittest
+
+class TestCApiSymbols(unittest.TestCase):
```

Comment:
Can you add a docstring ?

Reply:
Done, PTAL

Reply:
Ah well, and apparently also for the module according to some strict linter somewhere :)